scripts/deploy-dao.ts

```diff
@@ -0,0 +1,135 @@
+    // azoriusTxBuilder.buildUpdateDAOSnapshotURLTx(),
+    azoriusTxBuilder.buildLinearVotingContractSetupTx(),
+    azoriusTxBuilder.buildEnableAzoriusModuleTx(),
+    azoriusTxBuilder.buildAddAzoriusContractAsOwnerTx(),
+    azoriusTxBuilder.buildRemoveMultiSendOwnerTx(),
+  ];
+  console.log("Internal txs created");
+
+  txs.push(azoriusTxBuilder.buildDeployStrategyTx());
+  txs.push(azoriusTxBuilder.buildDeployAzoriusTx());
+  txs.push(
+    azoriusTxBuilder.buildExecInternalSafeTx(
+      azoriusTxBuilder.signatures(),
+      internalTxs
+    )
+  );
+  console.log("Tx created");
+
+  const encodedTx = encodeMultiSend(txs);
+
+  console.time("Tx sent");
+  const execution = await multisendContract.multiSend(encodedTx, {
+    gasLimit: 5000000,
+  });
+  execution.wait();
+  console.timeEnd(`tx executed ${execution.hash}`);
+
+  console.log(`
+O)))))                                          O))         O)))))          O)           O))))                O))                                O))                 O))
+O))   O))                                       O))         O))   O))      O) ))       O))    O))          O))   O))                             O))                 O))
+O))    O))   O))       O)))   O))    O)) O))  O)O) O)       O))    O))    O)  O))    O))        O))       O))       O) O)))   O))       O))    O)O) O)   O))         O))
+O))    O)) O)   O))  O))    O)   O))  O))  O))  O))         O))    O))   O))   O))   O))        O))       O))        O))    O)   O))  O))  O))   O))   O)   O))  O)) O))
+O))    O))O))))) O))O))    O))))) O)) O))  O))  O))         O))    O))  O)))))) O))  O))        O))       O))        O))   O))))) O))O))   O))   O))  O))))) O))O)   O))
+O))   O)) O)         O))   O)         O))  O))  O))         O))   O))  O))       O))   O))     O))         O))   O)) O))   O)        O))   O))   O))  O)        O)   O))
+O)))))      O))))      O)))  O))))   O)))  O))   O))        O)))))    O))         O))    O))))               O))))  O)))     O))))     O)) O)))   O))   O))))    O)) O))
+  `);
+  console.table({ daoAddress: predictedSafeContract.address });
+
+  console.log("Transfering remaining balance of DCNT to Decent DAO");
+  const amountToTransfer = utils
+    .parseEther(decentDAOConfig.initialSupply)
+    .sub(totalLockedAmount);
+  const tokenTransfer = await dcntTokenContract.transfer(
+    predictedSafeContract.address,
+    amountToTransfer
+  );
+  tokenTransfer.wait();
+
+  console.log("Tokens transfered");
+  console.table({
+    amountToTransfer: amountToTransfer,
+    hash: tokenTransfer.hash,
+  });
+
+  console.log("Transfering token ownership to Decent DAO");
+  const transferTokenOwnership = await dcntTokenContract.transferOwnership(
+    predictedSafeContract.address
+  );
+  transferTokenOwnership.wait();
+  console.log("Token ownership transfered");
+  console.table({
+    dao: predictedSafeContract.address,
+    hash: transferTokenOwnership.hash,
+  });
+}
```

Comment:
These last two steps:

- Transfers remaining token balance from 'deployer' address to Decent DAO
- Transfers Token Ownership from 'deployer' to Decent DAO